packages/node/src/integrations/console.ts

```diff
@@ -14,11 +17,17 @@ const _consoleIntegration = (() => {
           return;
         }
 
+        // Truncate the breadcrumb length to max 2KB including the truncated snippet
+        let formattedMessage: string = util.format.apply(undefined, args);
+        if (formattedMessage.length > MAX_BREADCRUMB_MSG_LENGTH) {
```

Comment:
l: Why not just use `truncate` from `@sentry/utils`? :)

Reply:
I did not know that existed 🧠